packages/devextreme/js/__internal/grids/grid_core/columns_resizing_reordering/m_columns_resizing_reordering.ts

```diff
@@ -49,19 +50,23 @@ const allowReordering = function (that) {
   return that.option('allowColumnReordering') || that.getController('columns').isColumnOptionUsed('allowReordering');
 };
 
-const TrackerView = modules.View.inherit({
+class TrackerView extends modules.View {
+  private _positionChanged: any;
+
+  private _tablePositionController: any;
+
   _renderCore() {
-    const deferred = this.callBase();
+    const deferred = super._renderCore();
     this.element().addClass(this.addWidgetPrefix(TRACKER_CLASS));
     this.hide();
     return deferred;
-  },
+  }
 
   _unsubscribeFromCallback() {
     if (this._positionChanged) {
       this._tablePositionController.positionChanged.remove(this._positionChanged);
     }
-  },
+  }
 
   _subscribeToCallback() {
     const that = this;
```

Comment:
Let's replace ```that``` -> ```this``` if it's possible :)
(We can use the arrow function in the ```_positionChanged``` to prevent loosing context)